rasa/nlu/training_data/training_data.py

```diff
@@ -384,6 +415,16 @@ def validate(self) -> None:
                     f"this the training may fail."
                 )
 
+        # emit warnings for response intents without a response template
+        for example in self.training_examples:
+            if example.get(RESPONSE_KEY_ATTRIBUTE) and not example.get(RESPONSE):
+                raise_warning(
+                    f"Your training data contains an example '{example.text[:20]}...' "
```

Comment:
optional: use a constant for `20`

Reply:
I think it's fine to keep the 20: it is readable and just for logging. since it is very obvious what is happening, naming this random constant will not increase readability.